spec/components/parameters/files/limit.ts

```diff
@@ -0,0 +1,15 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../utils/ref';
+
+const limit: ParameterObject = {
+  name: 'limit',
+  in: 'query',
+  description: 'The maximum amount of files to be returned. (Min: 1 | Max: 10000)',
```

Comment:
tem o campo minimum and maximum no spec, aí não só aparece no doc sem precisar colocar manualmente na description, como também o validador já barra qualquer valor diferente disso.